t/plugin/limit-count-redis-cluster.t

```diff
@@ -328,7 +328,7 @@ code: 200
                             "limit-count": {
                                 "count": ]] .. count .. [[,
                                 "time_window": 60,
-                                "key": "http_x_real_ip",
+                                "key": "remote_addr",
```

Comment:
Can we change the relative tests to use remote_addr too?

Reply:
yes of course.